web/src/assets/licenses.json

```diff
@@ -0,0 +1,363 @@
+{
```

Comment:
I think this should be deleted. The licenses are already found in `web/assets/licenses.json` instead of `web/src/assets/licenses.json`